src/commonMain/kotlin/app/cash/turbine/Turbine.kt

```diff
@@ -94,7 +94,6 @@ public operator fun <T> Turbine<T>.plusAssign(value: T) { add(value) }
  * [withTurbineTimeout].
  * @param name If non-null, name is added to any exceptions thrown to help identify which [Turbine] failed.
  */
-@Suppress("FunctionName") // Interface constructor pattern.
```

Comment:
No es necesario?

Reply:
Yeah seems like the IDE finally learned this. It issues a warning that the suppression is unnecessary.